README.md

```diff
@@ -156,188 +173,188 @@ Note: you can evaluate message type from `messageType` parameter, based on that
 1. Enable and disable specific features with `FeatureActiveConfig`.
 ```dart
 ChatView(
-  ...
-  featureActiveConfig: FeatureActiveConfig(
-    enableSwipeToReply: true,
-    enableSwipeToSeeTime: false,
-  ),
-  ...
+...
+featureActiveConfig: FeatureActiveConfig(
+enableSwipeToReply: true,
+enableSwipeToSeeTime: false,
+),
+...
 )
 ```
 
 2. Adding an appbar with `ChatViewAppBar`.
 ```dart
 ChatView(
-  ...
-  appBar: ChatViewAppBar(
-    profilePicture: profileImage,
-    chatTitle: "Simform",
-    userStatus: "online",
-    actions: [
-      Icon(Icons.more_vert),
-    ],
-  ),
-  ...
+...
+appBar: ChatViewAppBar(
+profilePicture: profileImage,
+chatTitle: "Simform",
+userStatus: "online",
+actions: [
+Icon(Icons.more_vert),
+],
+),
+...
 )
 ```
 
 3. Adding a message list configuration with `ChatBackgroundConfiguration` class.
 ```dart
 ChatView(
-  ...
-  chatBackgroundConfig: ChatBackgroundConfiguration(
-    backgroundColor: Colors.white,
-    backgroundImage: backgroundImage,
-  ),
-  ...
+...
+chatBackgroundConfig: ChatBackgroundConfiguration(
+backgroundColor: Colors.white,
+backgroundImage: backgroundImage,
+),
+...
 )
 ```
 
 4. Adding a send message configuration with `SendMessageConfiguration` class.
 ```dart
 ChatView(
-  ...
-  sendMessageConfig: SendMessageConfiguration(
-    replyMessageColor: Colors.grey,
-    replyDialogColor:Colors.blue,
-    replyTitleColor: Colors.black,
-    closeIconColor: Colors.black,
-  ),
-  ...
```

Comment:
Did you remove then indentation here and in the other places of the README by accident?